Ruqoyah/SELA
# #157332433 Create landing page front end

#### What does this PR do?
This PR designs the landing page of the application

#### Description of Task to be completed?
This task designs the landing page of the application.

#### How should this be manually tested?
- Clone the repo
- Run `npm install`
- Start the application using `localhost:8000` then you should be able to see the frontend design

#### What are the relevant pivotal tracker stories?
[157332433](https://www.pivotaltracker.com/story/show/157332433)

#### Any background context you want to add?
N/A

#### Screenshots
<img width="1440" alt="screen shot 2018-05-09 at 11 08 44 am" src="https://user-images.githubusercontent.com/24945610/39809748-9e48008e-537a-11e8-82ae-fa4cde4edc40.png">
<img width="1440" alt="screen shot 2018-05-09 at 11 08 56 am" src="https://user-images.githubusercontent.com/24945610/39809399-923195d6-5379-11e8-94c2-9bc1c8b447e3.png">
<img width="1438" alt="screen shot 2018-05-09 at 11 09 05 am" src="https://user-images.githubusercontent.com/24945610/39809368-7e9e1e86-5379-11e8-9b3d-e2d659d65921.png">
<img width="1440" alt="screen shot 2018-05-09 at 11 09 24 am" src="https://user-images.githubusercontent.com/24945610/39809367-7e768f06-5379-11e8-8a45-1340e5df5db4.png">
